kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -1096,7 +1096,7 @@ private CompletableFuture<Pair<Errors, Long>> fetchOffset(String topicName, long
         CompletableFuture<Pair<Errors, Long>> partitionData = new CompletableFuture<>();
 
         topicManager.getTopic(topicName).thenAccept((perTopicOpt) -> {
-            if (!perTopicOpt.isPresent()) {
+            if (perTopicOpt.isEmpty()) {
```

Comment:
The `Optional#isEmpty` is introduced in JDK11. Do we need cherry-pick this PR to 2.9.3 or 2.10.1? Since they are using JDK1.8 to run the tests.